### ISSUE TITLE
Short middle leg for lower case m?

### ISSUE THREAD

**Issue by AndydeCleyre** [0xType/0xProto#65]
Hi! Thanks for this, it's fantastic.

Probably the current design for lower case `m` is very deliberate and not to change, but I thought I'd ask anyway.

Have you considered shortening the middle leg of the `m`? To me, especially at small sizes, that really helps to give the glyph its own shape, and avoids looking like a dense block.

FWIW here are the other coding fonts I can find featuring a short middle for the lower case `m`:

Ubuntu Mono: 
![image](https://github.com/0xType/0xProto/assets/1787385/45a4a859-204b-49e3-b1ad-5144bfdfb134)

Iosevka: 
![image](https://github.com/0xType/0xProto/assets/1787385/d82b77e0-9786-4b92-8ed3-95bf13d540f2)

BigBlue Terminal: 
![image](https://github.com/0xType/0xProto/assets/1787385/24abfa63-f7ab-4cac-b40a-85f96e251603)

Binchotan Sharp: 
![image](https://github.com/0xType/0xProto/assets/1787385/a443f533-4c77-4b39-966f-54ce0451886a)

Fixedsys: 
![image](https://github.com/0xType/0xProto/assets/1787385/1791e969-55b2-4ef2-89ea-19f00fc7b04b)

Lekton: 
![image](https://github.com/0xType/0xProto/assets/1787385/d3075069-5dd3-4f97-a09d-4e50fa2b35f1)

Monaspace Radon: 
![image](https://github.com/0xType/0xProto/assets/1787385/3fb9b0bc-1a1a-4a4e-a37d-043ec14e07c0)

Monocraft: 
![image](https://github.com/0xType/0xProto/assets/1787385/7c387756-300c-484b-b025-fb4bec686745)

Monoflow: 
![image](https://github.com/0xType/0xProto/assets/1787385/d04deb49-407a-430a-ae11-5778d2d2e76b)

Monofoki: 
![image](https://github.com/0xType/0xProto/assets/1787385/bfe4dea5-5ea3-45cc-8ad6-8572c8c76c86)

Scientifica: 
![image](https://github.com/0xType/0xProto/assets/1787385/14e2bcb1-3505-4698-ba0d-d424283b32db)

Spleen: 
![image](https://github.com/0xType/0xProto/assets/1787385/321541cd-d26c-426f-b5c0-9604e0b41d0e)

**Comment by masaakim** [comment]
Hi @AndydeCleyre , thank you for using!

I knew of fonts that used a short middle leg m like the one you gave as an example, but thought it might not fit 0xProto.

But you are right, it looks good for legibility, especially at small sizes.
I will actually make a glyph and test it before deciding whether to change it or not.

I'm on vacation until April 17, so I'll do that after that :)

**Comment by masaakim** [comment]
I shortened the middle leg of the lowercase `m`. This is the new lowercase glyph for 0xProto.

Compared to Ubuntu Mono, the middle leg may still seem long, but if I shortened it too much, only `m` would appear to float above the baseline, changing the rhythm of the other letters and making it difficult to read, so I went with the current length.

<img width="1610" alt="image" src="https://github.com/0xType/0xProto/assets/1401513/7cddc66c-9067-479b-ac5f-53186ef85205">

This change will be included in the v2.000 ( https://github.com/0xType/0xProto/pull/66 ) release.
